### DISCUSSION TITLE
🚀 Enhance cache management with cache tags

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4248] [Feature Requests]
### Suggest an idea for ExpressionEngine

Currently, when an entry is saved, caches are cleared for the entire EE website, which is not optimal. It would be great to have something like cache tags to specify which caches should be cleared.

I'm thinking of attribute "cache_tags" for {exp:channel:entries}, like this:
```
{exp:channel:entries 
    channel="news"
    cache="yes"
    refresh="3600"
    cache_tags="news|index_page|news_listings"
}
```
And a similar ‘Cache Tags’ field in the template settings, if a user prefers to use template caching.

Also, this would require per-channel settings for cache clearing when there's a new or updated entry. Like, "when a new entry is saved, clear these cache tags, and when an entry is updated, clear these cache tags".

**Comment by intoeetive** [comment]
Not exactly what you're requesting here, but you can use cache prefix : https://docs.expressionengine.com/latest/channels/entries.html#cache-refresh-cache_prefix

Clearing the caches is always global though, irrespective of prefix
